docs/algorithms/sig/sphincs.md

```diff
@@ -66,7 +66,7 @@ Are implementations chosen based on runtime CPU feature detection? **Yes**.
 |       Implementation source       | Identifier in upstream   | Supported architecture(s)   | Supported operating system(s)   | CPU extension(s) used   | No branching-on-secrets claimed?   | No branching-on-secrets checked by valgrind?   | Large stack usage?   |
 |:---------------------------------:|:-------------------------|:----------------------------|:--------------------------------|:------------------------|:-----------------------------------|:-----------------------------------------------|:---------------------|
 | [Primary Source](#primary-source) | clean                    | All                         | All                             | None                    | True                               | True                                           | False                |
-| [Primary Source](#primary-source) | avx2                     | x86\_64                     | All                             | AVX2                    | True                               | True                                           | False                |
+| [Primary Source](#primary-source) | avx2                     | x86\_64                     | Linux,Darwin                    | AVX2                    | True                               | True                                           | False                |
```

Comment:
So optimized Sphincs code is no longer supported on Windows? Does anyone know why? Looks like a regression, no?

Reply:
That's weird... there was no corresponding change in PQClean, where we (presumably?) pulled this data from. I'll take a look at the `copy_from_upstream` script and see if I can figure out what's going on.

Reply:
[Link to the relevant file](https://github.com/PQClean/PQClean/blob/8e220a87308154d48fdfac40abbb191ac7fce06a/crypto_sign/sphincs-sha2-192s-simple/META.yml)

Reply:
Never mind, this seems to be caused by #1478. Not sure why it didn't get updated in previous pulls from upstream, though.

Reply:
> Not sure why it didn't get updated in previous pulls from upstream, though.

Don't understand this either. Nor why I didn't catch/check the effect of #1478 to documentation then :-( But then it's now "fixed".